Dacaspex/Exhibit
# #366 Clickable configuration rows

#### What have I done
Made rows in the configuration menu clickable which toggle the checkbox.

#### Testing
1. Go to configuration.
2. Click on a configuration row and observe that the checkbox toggles.

#### Addresses SRD section(s)
None
